modules/chronograf-server/variables.tf

```diff
@@ -119,34 +137,38 @@ variable "ebs_block_devices" {
 
 variable "wait_for_capacity_timeout" {
   description = "A maximum duration that Terraform should wait for ASG instances to be healthy before timing out. Setting this to '0' causes Terraform to skip all Capacity Waiting behavior."
+  type        = string
   default     = "10m"
 }
 
 variable "health_check_type" {
   description = "Controls how health checking is done. Must be one of EC2 or ELB."
+  type        = string
   default     = "EC2"
 }
 
 variable "health_check_grace_period" {
   description = "Time, in seconds, after instance comes into service before checking health."
+  type        = number
   default     = 600
 }
 
 variable "instance_profile_path" {
   description = "Path in which to create the IAM instance profile."
+  type        = string
   default     = "/"
 }
 
 variable "ssh_port" {
   description = "The port used for SSH connections"
+  type        = number
   default     = 22
 }
 
 variable "tags" {
   description = "List of extra tag blocks added to the auto scaling group configuration. Each element in the list is a map containing keys 'key', 'value', and 'propagate_at_launch' mapped to the respective values."
-  type        = "list"
+  type        = any
```

Comment:
We can probably narrow the type down here since the inner `map` has well defined keys:

```
  type = list(object({
    key = string
    value = string
    propagate_at_launch = bool
  }))
```

(Intentionally not a suggestion, as this is not correctly formatted with `terraform fmt`)